conan/tools/microsoft/visual.py

```diff
@@ -268,3 +268,14 @@ def is_msvc_static_runtime(conanfile):
     :return: True, if msvc + runtime MT. Otherwise, False
     """
     return is_msvc(conanfile) and "MT" in msvc_runtime_flag(conanfile)
+
+
+def msbuild_arch(conanfile):
+    if conanfile.settings.get_safe("os") == "WindowsCE":
+        return conanfile.settings.get_safe("os.platform")
```

Comment:
Those 2 lines are new logic not previously in the code, and those are untested. Better leave them as-is, and if necessary, treat the problem of WindowsCE in a different PR.

Reply:
reverted